execute.py

```diff
@@ -491,6 +500,7 @@ def OU(v):
             #####################################
 
             # V step
+            V_stupid(v)
```

Comment:
Does this need to be prefaced with `v = `?

Reply:
no, it is purely for the sake of logging...if the script is run, it will print a `stupid` ke from the incremental velocity update and the ke of the `real` incremental velocity...eyeballing them gives the exact same ke...

Reply:
Okay! I'm not 100% sure what's being tested by the new definition -- I had meant to suggest trying to simplify to a single scalar mass for the purpose of debugging.

I think I had misunderstood a lot here. I had read the code assuming mistakenly that mass_matrix was a square matrix, but just realized it is defined as an (n_particles, 1) vector. (So each appearance of `mass_matrix[0,:][:,None]` could be replaced with `mass_matrix.T` if you want.) Even better, could we rename this to `masses` and have it be of shape `(n_particles, )` instead?

Reply:
i can't take `mass_matrix.T` without losing the mass units, so i use `mass_matrix[0,:][:,None]`...but all of this is consistent...based on the last test and the two logger statement reduced incremental kinetic energies, i am less than convinced that the problem is here...

Reply:
Okay! Just a readability thing. ("mass matrix" usually refers to a square matrix, so I didn't realize it was a vector until I read the point where it was defined -- if there are just n_particles numbers could use the name "masses" or similar)

Reply:
i had originally named it mass matrix because i was convinced i was going to use an nxn diagonal matrix to define the masses and do algebra in the integrator, but performing matrix multiplication with it proved a little slow, so i switched it to a vector and left the name.  regardless, if you look through the logger comments, yo u can see the loop that is running the VRORV integrations between incrementing the lambda value.  the entire reason i thought that there was a bug here somewhere is because i noticed that the kinetic energy of the system seems to decrease rather dramatically and then level out somewhere toward the end of the annealing process if i am _not_ in the overdamped regime. it may be worthwhile to look at the logger comments that are spat out of the `anneal` method.  there is also a nonzero chance that there isn't a bug and i'm wrong

Reply:
Okay! I took a brief look at the log file but will need more time to understand what I'm looking at -- so far just noticed that KE by end of trajectory is much lower than the Maxwell-Boltzmann sampled KE...